e2e/send_tex_address.test.js

```diff
@@ -0,0 +1,38 @@
+const { log, device, by, element, expect } = require('detox');
+
+import { loadTestWallet } from './e2e-utils/loadTestWallet.js';
+
+const sleep = ms => new Promise(r => setTimeout(r, ms));
+
+describe('Renders wallet data correctly.', () => {
+  it('loads a wallet', loadTestWallet);
+  it('parses the TEX address and correctly renders the confirm screen', async () => {
+    await waitFor(element(by.id('vt-1')))
+      .toExist()
+      .withTimeout(20000);
+    await element(by.text('SEND')).tap();
+
+    // Address taken from the reference implementation
+    await element(by.id('send.addressplaceholder')).replaceText('tex1s2rt77ggv6q989lr49rkgzmh5slsksa9khdgte');
+    await element(by.id('send.amount')).replaceText('0');
+    await element(by.id('send.checkboxua')).tap();
```

Comment:
I don't think the TEX addresses will accept `memos`, because somehow they are still T addresses. You need to remove all lines referring to the memo field, that will be hide all the time.